en/concept/mixing.md

```diff
@@ -236,3 +236,33 @@ Idlespeed can range from 0.0 to 1.0. Idlespeed is relative to the maximum speed
 
 In the case where an actuator saturates, all actuator values are rescaled so that 
 the saturating actuator is limited to 1.0.
+
+#### Helicopter Mixer
+
+The helicopter mixer combines three control inputs (roll, pitch, thrust) into four outputs ( swash-plate servos and main motor ESC setting). The first output of the helicopter mixer is the throttle setting for the main motor. The subsequent outputs are the swash-plate servos. The tail-rotor can be controlled by adding a simple mixer.
+
+The thrust control input is used for both the main motor setting as well as the collective pitch for the swash-plate. It uses a throttle-curve and a pitch-curve, both consisting of five points. This allows for more complex behavior.
+
+The mixer definition begins with:
+
+```
+H: <number of swash-plate servos, either 3 or 4>
+T: <throttle setting at 0% thrust> <25%> <50%> <75%> <100%>
+P: <collective pitch at 0% thrust> <25%> <50%> <75%> <100%>
+```
+This is followed by either 3 or 4 lines for the swash-plate servos in the following form:
+```
+S: <angle> <arm length> <scale> <offset> <lower limit> <upper limit>
+```
+
+The throttle- and pitch-curves define five points in the range between -10000 to +10000. For simple linear behavior, the five values for a curve should be `-10000 -5000 0 5000 10000`.
```

Comment:
Because the line above this paragraph is the `S` line, my assumption here is that the `S` line is what defines a "throttle and pitch curve". Is that right?
What is confusing here is that the S line has 6 fields, but you're talking about only 5 values. Also you have 3 or 4 s lines but you talk about a throttle curve and a pitch curve and it isn't clear what the mapping between swash plate servos and the curve are - ie does each servo have a curve, does it have both curves?

Anyway, assuming this does apply to the S line, I would move this text down below the following paragraph and explain what the values of the curve map to. I might also show two graphs of the curves, showing linear and more complicated behaviour.